designs/virtualization/4.x/create-vm-dropdowns/create-vm-dropdowns.md

```diff
@@ -0,0 +1,137 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# create VM dropdowns
+
+This item will cover various updates to the VM creation wizard
+- Adding descriptions to the dropdown fields (using this pattern https://github.com/patternfly/patternfly-design-kit/issues/218)
+- Renaming fields
+- Reordering fields
+- Adding 'Operating System source' step
+
+
+![Create VM - General-default](img/1-0.jpg)
+Landing on the General step, Default mode.
+Section titles added, and fields reorgenized and renamed.
+Any OS image source related fields are mofed to the next step.
+
+'General' section:
+- 'Name'
+- 'Description'
+'Resources & configuration' section'
+- 'Operating system'
+- 'Workload type' (previously Workload profile)
+- 'Flavor (CPU and Memory)' (previously only 'flavor')
+
+
+## General step
+
+![Create VM - General](img/1-1.jpg)
```

Comment:
nit - in the flavor dropdown value it has `2 CPC, 16 Bip`

Reply:
Still seeing `Bip` for memory :(

Reply:
The Zombie BIP. I've updated 1-1 and 1-1-2 jpgs.